_index.scss

```diff
@@ -2,6 +2,3 @@
 	gap-class-selectors,
 	attribute-selectors,
 	gap-attribute-selectors;
-
-@forward "./src/lib/mixins" hide layout-gap-attributes,
-	layout-gap-classes;
```

Comment:
This removes all mixins from the public API. We should keep the following in the file:
```scss
@forward "./src/lib/mixins";
```

Reply:
I'll add it again.